setup.py

```diff
@@ -10,4 +10,7 @@
     author_email="opensource@yelp.com",
     description="A tool for answering \"How'd that malware get there?\"",
     packages=["osxcollector"],
+    install_requires=[
```

Comment:
For install_requires we're only including what's required for collection.
I think we only need `threat_intel` for analysis.